.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: OSV PR format check
+
+on:
+  pull_request:
```

Comment:
Hmm looking at the documentation https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#event-triggers, does this need to be `pull_request_target` instead?

Reply:
Maybe I'm wrong, but I think `pull_request` is sufficient and better for us. Considering it only provides [read-only permissions on forked repos](https://github.blog/changelog/2021-02-19-github-actions-workflows-triggered-by-dependabot-prs-will-run-with-read-only-permissions/), so it's safer (and we only need read permissions).

I think that GitHub Actions has features for editing pull requests, so they recommend using `pull_request_target `to get full permissions (including writing).

(My this PR is merging from a forked repo and has successfully passed the title check action, so I think it should be fine.)

Reply:
Ah that makes sense, can you add a comment here saying pull_request_target is only necessary for editing just to remind us in the future? Otherwise LGTM

Reply:
Updated, thanks!